content/en/agent/guide/network.md

```diff
@@ -52,6 +52,13 @@ All Agent traffic is sent over SSL. The destination is dependent on the Datadog
 : `rum.`{{< region-param key="browser_sdk_endpoint_domain" code="true" >}}<br>
 `session-replay.`{{< region-param key="browser_sdk_endpoint_domain" code="true" >}}
 
+{{< site-region region="us,eu,us3,us5,ap1" >}}
+[Remote Configuration][1]
+: `config.`{{< region-param key="dd_site" code="true" >}}
```

Comment:
Is the top-level `config` key correct here? I would assume there's a specific subkey that needs changing.

Reply:
Yes, I believe "config" is the top-level key and then the domain will change based on site being used, e.g., 

US1: config.datadoghq.com
US3: config.us3.datadoghq.com
US5: config.us5.datadoghq.com
EU: config.datadoghq.eu
AP1: config.ap1.datadoghq.com

@arbll can you please confirm?